ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign-v4-simulation/value-display/value-display.tsx

```diff
@@ -31,6 +31,9 @@ import {
 import { TOKEN_VALUE_UNLIMITED_THRESHOLD } from '../../../shared/constants';
 import { getAmountColors } from '../../../utils';
 
+export const DAI_CONTRACT_ADDRESS =
```

Comment:
Minor, is this better in `ui/pages/confirmations/components/confirm/info/shared/constants.ts`?